data/phoible-references.bib

```diff
@@ -42457,3 +42457,23 @@ @book{heath2015togokan
 	Title = {A Grammar of Togo Kan (Dogon language family, Mali)},
 	Url = {https://deepblue.lib.umich.edu/handle/2027.42/123066},
 	Year = {2015}}
+
+@article{rose2021mojenotrinitario.pdf,
+	title={Mojeño Trinitario},
+	volume={52},
+	DOI={10.1017/S0025100320000365},
+	number={3},
+	journal={Journal of the International Phonetic Association},
+	publisher={Cambridge University Press},
+	author={Rose, Françoise},
+	year={2022},
+	pages={562–580}}
+
+@article{guzzo2022talian.pdf,
+	title={Brazilian Veneto (Talian)},
```

Comment:
@Alessioryan -- articles titles will also get automatically lower-cased, so they proper nouns like language names need to be escaped, e.g., {Brazilian Veneto (Talian)} --> {{Brazilian Veneto (Talian)}}. Please update.

Reply:
Makes sense, I just updated it. Thanks!